src/services/File Management/server.js

```diff
@@ -0,0 +1,19 @@
+const express = require("express");
+const fs = require("fs");
+require("dotenv").config();
+const app = express();
+app.use(express.json());
+const fileRoutes = require("./routes/filesRoutes");
+
+app.use("/api/file", fileRoutes);
+
+app.listen(process.env.PORT, () => {
+  console.log("[!] server is running on port:", process.env.PORT);
```

Comment:
replace `process.env.PORT` with `process.env.PORT || 3000` in case the `PORT` wasn't supplied in the `.env` file.